eurofaang_ri_be/users/serializers.py

```diff
@@ -3,8 +3,18 @@
 
 
 class UserSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    username = serializers.CharField(required=True, allow_blank=False)
+    first_name = serializers.CharField(required=True, allow_blank=False)
+    last_name = serializers.CharField(required=True, allow_blank=False)
+    email = serializers.CharField(required=True, allow_blank=False)
+    phone_number = serializers.CharField(required=True, allow_blank=False)
+    organization_name = serializers.CharField(required=True, allow_blank=False)
+    organization_address = serializers.CharField(required=True, allow_blank=False)
+    role = serializers.CharField(required=True, allow_blank=False)
```

Comment:
all fields must be optional, as we want user to save form and return back to complete it